include/seqan3/core/algorithm/concept.hpp

```diff
@@ -42,13 +42,13 @@ namespace seqan3::detail
  *                           this table to allow validation checks.
  */
 template <typename algorithm_id_type>
-inline constexpr std::array<std::array<void, 0>, 0> compatibility_table;
+inline constexpr std::array<std::array<void *, 0>, 0> compatibility_table{};
```

Comment:
Why is it now void * instead of void?

Reply:
void is an invalid value type for std::array since [std::array](https://en.cppreference.com/w/cpp/container/array) uses `reference` which is defined as `value_type&`. But, `void &` is no valid type.

Reply:
ah ok. thanks for the explanation :)